src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/BasicLoadBalancingPolicyQueryPlanTest.java

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
This class is [already defined](https://github.com/apache/cassandra-java-driver/blob/4.18.1/core/src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/BasicLoadBalancingPolicyQueryPlanTest.java) in cassandra-java-driver... why do we also need it here?  Based on a diff between the two they're basically identical.